Okta.Core.Tests/Clients/Helpers.cs

```diff
@@ -155,5 +156,31 @@ internal static Models.Group CreateGroup(OktaClient oktaClient, TestGroup dbGrou
             return oktaGroup;
         }
 
+        internal static string GetRandomString()
+        {
+            const string AllowedChars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz#@$^*()";

```

Comment:
lower case of `allowedChars` within a method,  unless you want to make it a static field